packages/@lwc/shared/src/api-version.ts

```diff
@@ -10,15 +10,20 @@ import { isNumber } from './language';
 export const enum APIVersion {
     V58_244_SUMMER_23 = 58,
     V59_246_WINTER_24 = 59,
+    V60_248_SPRING_24 = 60,
 }
 
 // These must be updated when the enum is updated.
 // It's a bit annoying to do have to do this manually, but this makes the file tree-shakeable,
 // passing the `verify-treeshakeable.js` test.
 
 export const LOWEST_API_VERSION = APIVersion.V58_244_SUMMER_23;
-export const HIGHEST_API_VERSION = APIVersion.V59_246_WINTER_24;
-const allVersions = [APIVersion.V58_244_SUMMER_23, APIVersion.V59_246_WINTER_24];
+export const HIGHEST_API_VERSION = APIVersion.V60_248_SPRING_24;
+const allVersions = [
+    APIVersion.V58_244_SUMMER_23,
+    APIVersion.V59_246_WINTER_24,
+    APIVersion.V60_248_SPRING_24,
+];
```

Comment:
We should add a step in the post-CLCO doc to update this too (or add a script to automate it).

Reply:
I figured we would only need to update this when we add some feature to API versioning. It's possible that we'll end up skipping a release, e.g. because we didn't do any API versioning changes that release. So I don't think this necessarily need to be updated every release.